### DISCUSSION TITLE
Collection Group

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2405] [Q&A]
What do you think if we can group collections?

**Comment by hungcrush** [comment]
My Idea: 

![CleanShot 2023-05-02 at 17 21 26](https://user-images.githubusercontent.com/8098916/235641429-71798aba-072a-41a2-8aeb-94f622be0fed.jpg)

**Reply by ganigeorgiev** [reply]
I have no strong opinion about this. My only concern is that it may clutter the UI. Other users also suggested something similar or option to allow reordering but for now I'm not planning working on this since it may require some major internal changes and I don't think it is worth the dev effort at the moment. It is in my todo to research and test the various options, but remains a very low priority.

**Reply by hungcrush** [reply]
I got it.
